new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItSimpleNginxValidation.java

```diff
@@ -628,4 +673,149 @@ private void checkServiceCreated(String serviceName) {
 
   }
 
+  /**
+   * Check pod does not exist.
+   *
+   * @param podName pod name to check
+   */
+  private void checkPodDoesNotExist(String podName) {
```

Comment:
method name does not seem to match what the method actually does

Reply:
changed it to checkPodDeleted(String podName)